docs/help/git_webhooks.md

```diff
@@ -0,0 +1,100 @@
+already [setup it](https://github.com/Neilpang/acme.sh/blob/master/README.md),
+you can install a certificate for bitbot by: `/root/.acme.sh/acme.sh --install-cert -d <domain> --key-file /somewhere/bitbot/owns/key.pem --fullchain-file /somewhere/bitbot/owns/cert.pem`.
+
+***WARNING!*** Always ensure that other users won't be able to read your
+keys. `chmod -R 700 /path/to/directory/containing/<key.pem-and-cert.pem> &&  chown -R bitbotaccount:bitbotgroup /path/to/directory/containing/<key.pem-and-cert.pem>`
+
+### Configure the bot
+
+1. Enable the REST API in bot.conf by adjusting
+
+```
+# key/cert used for REST API
+tls-api-key              = /path/to/key.pem
+tls-api-certificate      = /path/to/cert.pem
+api-port                 = 5000
+```
+
+you may change the API port if you wish.
+
+2. restart the bot or send it a `SIGUSR1` signal
+
+3. enable the REST API by `!config bot rest-api on`
+
+4. start the REST API for the first time by `!reloadmodule rest_api`
+
+5. create a new API key by `/msg <bot> apikey <apikey-name> /api/<github|gitea>`
+   * The `apikey-name` is how you can later revoke it in case of trouble.
+
+6. the bot will reply `[APIKey] New API key ('<apikey-name>'): <API-key>`.
+   Keep the `<API-key>` at hand, you will need it in the next part.
+
+7. add the repository to your channel with
+   `!webhook add <username|organization>/<reponame>`. You can
+   see the announced webhooks by `!webhook list` or remove them by
+   `!webhook remove <name>`
+    * you may `!webhook add <username|orgname>` to accept all repositories
+      from the user or organization.
+
+### Configure the git host
+
+This is generally done within settings of your repository and may depend
+on the Git host. The details you need are:
+
+* Target URL: `https://<your-bot-address>:<your-API-port>/api/<github|gitea>?key=<API-key>`
+    * `<github|gitea>` means that you either enter `github` if you use
+      GitHub or `gitea` if you use Gitea.
+    * the `<API-key>` is the same the bot gave you in the previous
+      step 6.
+* HTTP Method: POST
+* POST Content Type: `application/json` *or* `application/x-www-form-urlencoded`,
+  bitbot supports both of them.
+
+### Managing API keys
+
+The API key isn't tied up to a specific git repository. It's recommended
+that every user has their own API key.
+
+This isn't implemented yet, see [issue #123](https://github.com/jesopo/bitbot/issues/123).
+
+### Additional configuration
+
+There are several options that can be configured
+
+* on the bot level:
+    * `!config bot git-show-private on` to also show private git
+      repositories. They are hidden by default to avoid accidental leakage
+      of private data.
+* on the channel level:
+    * `!config channel auto-github-cooldown on` to ???
+    * `!config channel auto-github on` to ???
+    * `!config channel github-default-repo <user|orgname/repo>` in order to
+      have the bot link `<#issuenumber>` messages into the issues with
+      details on them.
```

Comment:
just guessing. Also is this GitHub specific? I am currently mainly a Gitea user with Bitbot.